nemo/collections/cv/modules/resnet.py

```diff
@@ -0,0 +1,159 @@
+from torch import nn
+
+from nemo.collections.cv.modules import Conv2d, GlobalAvgPool2d, ResNetBlock, ResNetBottleneckBlock
+from nemo.core.classes.module import NeuralModule
+
+
+class ResNet(NeuralModule):
```

Comment:
Should have neural types

Reply:
@titu1994  Should we keep the resnet in modules? The module is used in the ResNetVideoFrontEnd class.
I moved it to cv/parts/submodules/ for now